packages/blockchain-wallet-v4-frontend/src/data/modules/coinify/sagas.js

```diff
@@ -54,27 +59,60 @@ export default ({ coreSagas }) => {
     }
   }
 
-  const sell = function * (payload) {
+  const sell = function * () {
```

Comment:
Looks like the `fee` is missing. Here's the SFOX version: https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/data/modules/sfox/sagas.js#L124

Reply:
it can be hardcoded to `priority`

Reply:
No it's not missing, it's done via `this.props.sendBtcActions.sendBtcInitialized({ feeType: 'priority' })` in blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/Sell/index.js

Reply:
So the payment object we get with the getPayment selector already contains the priority fee when the saga is called

Reply:
nice!